index.html

```diff
@@ -11,58 +11,69 @@
 </head>
 </html>
 <body>
-    <div class="toast-container position-fixed bottom-0 end-0 p-3">
-        <div id="game-message" class="toast align-items-center text-bg-primary border-0" role="alert" aria-live="assertive" aria-atomic="true">
-          <div class="d-flex">
+    <div id="gamearea">
+        <div id="gamearea-popup" class="toast" role="alert">
             <div class="toast-body">
-                <a id="current-message"></a>
+                Welcome to the craps table!
```

Comment:
[nit] Perhaps `Craps Table` would be better since this is a proper noun? What do you think?

Reply:
I agree; made name change